addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -59,12 +59,13 @@
                     t-on-click.prevent="onBreadcrumbClicked(breadcrumb.jsId)"
                     >
                     <a href="#">
-                        <t t-esc="breadcrumb.name"/>
+                        <t t-if="breadcrumb.name" t-esc="breadcrumb.name"/>
+                        <t t-else="">Unnamed</t>
                     </a>
                 </li>
             </t>
-            <li class="breadcrumb-item active">
-                <t t-if="env.config.displayName" t-esc="env.config.displayName" />
+            <li t-if="env.config.getDisplayName" class="breadcrumb-item active">
```

Comment:
should this t-if be set on the `<ol class=breadcrumb">` node? What's the point of showing the breadcrumb if the current controller has no name?

Reply:
alternative to discuss: instead of providing `config.breadcrumbs` which is an array of breadcrumbs parts for the n-1 first controllers in the stack, what about providing `config.getBreadcrumbs`, a function that returns the breadcrumbs parts for all controllers in the stack? Then, we would do `<ol t-if="env.config.getBreadcrumbs" class="breadcrumb">`.